cloud_functions/climateiq_data_export_cf/main_test.py

```diff
@@ -46,11 +46,36 @@ def test_export_model_predictions_missing_study_area(
     }
     event = CloudEvent(attributes, data)
 
-    mock_study_area = mock.Mock()
-    mock_study_area.exists = False  # Indicate study area doesn't exist
+    # Build mock Storage object
+    mock_bucket = mock.create_autospec(storage.Bucket)
```

Comment:
Nit: This is reused across multiple tests so it may be worth pulling into its own function

Reply:
ack, the logic is much shorter now so going to leave as is